k8s/pkg/apis/longhorn/v1beta2/volume.go

```diff
@@ -99,6 +99,14 @@ const (
 	VolumeConditionReasonTooManySnapshots              = "TooManySnapshots"
 )
 
+type RestoreVolumeRecurringJobOpt string
```

Comment:
NIT: Just noticed that the builder validation is missing. But I don't know how to make the empty value as an enum option:
```
// +kubebuilder:validation:Enum=enabled;disabled
type RestoreVolumeRecurringJobOpt string
```